doc/release/upcoming_changes/template.rst

```diff
@@ -17,6 +17,7 @@
 {% if definitions[category]['showcontent'] %}
 {% for text, values in sections[section][category].items() %}
 {{ text }}
+
```

Comment:
I think you changed that for the release notes? I am not sure if we still want it or not.

Reply:
yes, I now added a release note that needs the blank line